### ISSUE TITLE
🐛a11y/useAltText: {...props} causes false positive

### ISSUE THREAD

**Issue by sevenc-nanashi** [biomejs/biome#3660]
### Environment information

```block
CLI:
  Version:                      1.7.3
  Color support:                true

Platform:
  CPU Architecture:             x86_64
  OS:                           linux

Environment:
  BIOME_LOG_DIR:                unset
  NO_COLOR:                     unset
  TERM:                         "xterm-256color"
  JS_RUNTIME_VERSION:           "v20.15.1"
  JS_RUNTIME_NAME:              "node"
  NODE_PACKAGE_MANAGER:         unset

Biome Configuration:
  Error:                        Found an unknown key `useImportExtensions`.
  Status:                       Loaded with errors
  Formatter disabled:           false
  Linter disabled:              false
  Organize imports disabled:    false
  VCS disabled:                 true

Workspace:
  Open Documents:               0
```

### What happened?

https://biomejs.dev/playground/?code=YwBvAG4AcwB0ACAATwBwAHQAaQBvAG4AYQBsAEkAbQBhAGcAZQAgAD0AIAAoAGEAbAB0ADoAIABzAHQAcgBpAG4AZwApACAAPQA%2BACAAewAKACAAIAByAGUAdAB1AHIAbgAgADwAaQBtAGcAIABhAGwAdAA9AHsAYQBsAHQAIAB8AHwAIAAiACIAfQAgAHsALgAuAC4AcAByAG8AcABzAH0AIAAvAD4ACgB9AAoACgA%3D

### Expected result

It should not warn the error.

### Code of Conduct

- [X] I agree to follow Biome's Code of Conduct

**Comment by ematipico** [comment]
That's intended by design.

Biome doesn't know if the props contain a prop that could pass an incorrect value for `alt`, so it's more conservative and it raises it as an issue.

You should flip the props or suppress the rule
